docker/webpack-compiler.entrypoint.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/bash
+# Copyright 2023 The Oppia Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+webpack_compiler_cmd="node"
+if [ "$prod_env" = "true" ]; then
+  webpack_compiler_cmd+=" --max_old_space_size=8192"
```

Comment:
if we set it from env variable, then it will be applicable always! we need this flag just when we are in `prod_env`. we cannot have this flag always because it allocates 8192 MB to the webpack compiler which is too bad in terms of resource allocation is we pass this always using env variables